src/Containers/Politicians/index.js

```diff
@@ -1,19 +1,45 @@
 /* @flow */
 import React from 'react';
 import PoliticianCard from '../../Components/PoliticianCard';
-import { politicians, promises } from '../../utils/testFixtures';
+import { getPoliticans } from '../../utils/api';
+import type { PoliticianType, PromiseType } from '../../types';
 
 export class Politicians extends React.Component {
+  state: {
+    politicians: Array<PoliticianType>,
+  };
+
+  state = {
+    politicians: [],
+  };
+
+  componentDidMount() {
+    getPoliticans().then(politicians => {
+      this.setState({ politicians });
+    });
+  }
+
+  promiseCompletionPercentage(promiseList: Array<PromiseType>): number {
```

Comment:
Thinking it might be nice if this was a utility function? I can see it being used again for political parties for example and maybe somewhere else as well?